doc/user/content/sql/create-source/mysql.md

```diff
@@ -259,7 +261,7 @@ The specified columns will be treated as `text`, and will thus not offer the
 expected MySQL type features. For any unsupported data types not listed above,
 use the [`EXCLUDE COLUMNS`](#excluding-columns) option.
 
-##### Truncation
+### Truncation
```

Comment:
I'm going to double-check this.  Was under the impression that subsources (i.e., tables replicated into Materialize) aren't writable (insert/update/delete) by the user.

Reply:
Oh ... heh heh ... I guess this meant truncated in the upstream side ... not within Materialize.

Reply:
This page will be updated as part of the source versioning docs work.